laravel/resources/js/ui/filter.js

```diff
@@ -59,4 +59,11 @@ $(document).ready(function()
     {
         $('.filter-departments').value(filterDepartment).trigger('change');
     }
+
+    //for the Admin User-list
+    $('.filter-user').on('change', function()
+    {
+        var role = $(this).value
```

Comment:
Why are you defining this unused variable?

Reply:
thats a good question, i'll get rid of it right away